webapp/tests/test_functions.py

```diff
@@ -3864,3 +3862,138 @@ def test_summarize_1minute_alignToFrom(self):
               },
               seriesList, "1minute", func, True)
           self.assertEqual(result, expectedResults[func])
+
+    def test_exponentialMovingAverage_integerWindowSize(self):
```

Comment:
cool.  I will refactor and commit.  It might take me until tonight (GMT-5), as I am at work now, etc.
Thanks,
Chris